src/test/java/com/microsoft/azure/batch/TaskTests.java

```diff
@@ -282,7 +282,12 @@ public void testAddMultiTasks() throws Exception {
             // LIST
             List<CloudTask> tasks = batchClient.taskOperations().listTasks(jobId);
             Assert.assertNotNull(tasks);
-            Assert.assertTrue(tasks.size() == 1000);
+            Assert.assertTrue(tasks.size() == TASK_COUNT);
+
+            // Test Job count
+            TaskCounts counts = batchClient.jobOperations().getTaskCounts(jobId);
+            int all = counts.active() + counts.completed() + counts.running();
```

Comment:
Assert on the `validationState` too?

Also might be helpful to have this in a different test than the `AddMultiTasks` one just for clarity/separation of concerns?